Unifying Kibana 9.0.0-rc1 RNs (#2994)
elastic/stack-docs: docs/en/install-upgrade/release-notes/release-notes-kibana.asciidoc

[[release-notes-kibana-9.0.0]]
= {kib} 9.0.0-rc1 release notes
++++
<titleabbrev>{kib}</titleabbrev>
++++

> For information about the {kib} 9.0.0 release, review the following information.

Reviewer: ```suggestion
Review the following information about the {kib} 9.0.0 release.
```

Reviewer: Just updating to the new template's phrasing

- `xpack.actions.rejectUnauthorized`
- `xpack.actions.proxyRejectUnauthorizedCertificates`
- `xpack.alerts.healthCheck`
- `xpack.alerts.invalidateApiKeysTask.interval`
- `xpack.alerts.invalidateApiKeysTask.removalDelay`
- `xpack.alerting.defaultRuleTaskTimeout`

For more information, refer to ({kibana-pull}198435[#198435]).
====

//Needs to be added to upgrade notes and detailed               
[discrete]
[[deprecation-208208]]
.Removed deprecated Cases APIs.
[%collapsible]
====
*Details* +
The following Cases APIs were removed:

- Get case status
- Get user actions
- Get all comments

For more information, refer to ({kibana-pull}208208[#208208]), ({kibana-pull}208086[#208086]), and ({kibana-pull}207926[#207926]).
====

//Needs to be added to upgrade notes and detailed
[discrete]
[[deprecation-203148]]
.Removed deprecated alerts routes.
[%collapsible]
====
*Details* +
The deprecated legacy alerts routes `api/alerts/alert` were removed. For more information, refer to ({kibana-pull}203148[#203148]).
====

**Discover and dashboards**

//Needs to be added to upgrade notes and detailed      
[discrete]
[[breaking-203927]]
.Disabled search sessions by default.
[%collapsible]
====

> *Details* +
> Search sessions are now disabled by default. For more information, refer to ({kibana-pull}203927[#203927]).

Reviewer 2: ```suggestion

Search sessions are now disabled by default. For more information, refer to (#203927).

*Details*

The button in Discover and Dashboards to create a new search session will no longer appear.

*Impact*

This impacts all users.

*Action*

To enable search sessions for all users, please set `data.search.sessions.enabled: true` in `kibana.yml`.

```

* If `savedQueryManagement` is `none`, the user cannot see or access the saved query management UI or APIs.
* If `savedQueryManagement` is `read`, the user can load queries from the UI and access read APIs, but cannot save queries from the UI or make changes to queries through APIs.
* If `savedQueryManagement` is `all`, the user can both load and save queries from the UI and through APIs.

*Action* +
Existing user roles that were previously implicitly granted access to saved queries through the dashboard, discover, visualize, or maps feature privileges will retain that access to prevent breaking changes. While no action is required for existing roles, it’s still advisable to audit relevant roles and re-save them to migrate to the latest privileges model. For new roles, ensure that the savedQueryManagement privilege is set as needed.
====

//Needs to be added to upgrade notes and detailed         
[discrete]
[[breaking-202679]]
.Removed `discover:searchFieldsFromSource` setting.
[%collapsible]
====

> *Details* +
> For more information, refer to ({kibana-pull}202679[#202679]).

Reviewer 2: ```suggestion
The `discover:searchFieldsFromSource` advanced setting has been removed.

*Details* +
Users can no longer configure Discover to load fields from `_source`. This is an internal optimization and should have little impact on users. https://github.com/elastic/kibana/pull/202679
```

====

//Needs to be added to upgrade notes and detailed           
[discrete]
[[breaking-201254]]
.Removed the legacy table in Discover.
[%collapsible]
====
*Details* +
It's no longer possible to use the legacy documents table in Discover. To that effect, the `doc_table:legacy` and `truncate:maxHeight` deprecated advanced settings have been removed. For more information, refer to ({kibana-pull}201254[#201254]).

> ====

Reviewer 2: ```suggestion

*Impact* +
This is primarily a UX change and should have little impact on Discover users.

====
```

[[release-notes-kibana-9.0.0]]
= {kib} 9.0.0-rc1 release notes
++++
<titleabbrev>{kib}</titleabbrev>
++++

Review the following information about the {kib} 9.0.0 release.

[float]
[[kibana-highlights-9.0.0]]
== Highlights

[[highlight-borealis]]
*New UI theme*. {kib} 9.0 introduces a more modern and refined look and feel. This new theme brings a vibrant color palette, improved dark mode support (including honoring your system preferences), and more that will bring your data in Kibana to life.

> 

Author: ```suggestion

[float]
[[kibana-known-issues-9.0.0]]
== Known issues

// tag::known-issue-211850[]
.Upgrade Assistant - Rollup jobs need to be stopped before rollup indices are reindexed.
[%collapsible]
====
*Details* +
Rollup indices, like all indices, created in 7.x or earlier need to be reindexed in preparation for migration to 9.0. However, in addition to the normal reindex process the rollup job also needs to be accounted for. It needs to be stopped before reindexing begins otherwise there may be a gap in rollup data. The job can be restarted when reindexing is complete.

This needs to be performed manually until addressed in the upgrade assistant code.

For more information, refer to https://github.com/elastic/kibana/issues/211850[#211850].
====
// end::known-issue-211850[]

```
Added as per https://github.com/elastic/kibana/pull/212637#discussion_r2004797009
